### ISSUE TITLE
Pattern Recognition Implementation docs.

### ISSUE THREAD

**Issue by dkgrif1286** [freqtrade/freqtrade#4340]
Does anyone know where I can find documentation that shows how to implement pattern recognition indicators? I can't find anything here or on freqtrade.io. Thanks!

**Comment by xmatthias** [comment]
Purely the calls are in the sample strategy / strategy templates. 

For more info - maybe try the [ta-lib docs](http://mrjbq7.github.io/ta-lib/func_groups/pattern_recognition.html) - however unfortunately they're not too good when it comes to "what does this return / how will the results look like" ...

**Comment by dkgrif1286** [comment]
Thanks @xmatthias! One final question.. hopefully. I got the bot to run but now I'm running into an error with importing 'merge_informative_pair' based on this documentation on freqtrade.io under the complete data-provider example: https://www.freqtrade.io/en/latest/strategy-customization/. every time I add from freqtrade.strategy.interface import IStrategy, merge_informative_pair, I get the error. 

![Error](https://user-images.githubusercontent.com/78570577/107302652-a3e94f80-6a4b-11eb-847f-2808b74dfe33.JPG)

I've attached my code as a txt because when I try to insert it, it messes up the first 15 lines.
[mfilong3.txt](https://github.com/freqtrade/freqtrade/files/5948076/mfilong3.txt)
